helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/net-exporter.json

```diff
@@ -528,7 +528,7 @@
   "schemaVersion": 20,
   "style": "dark",
   "tags": [
-    "owner:team-ludacris",
+    "owner:team-turtles",
```

Comment:
```suggestion
"owner:team-cabbage",
```

Reply:
I talked to Lucas before changing the ownership and he's not yet sure about net-exporter, because the metrics there are very much KaaS related. This could change with the cilium ownership now, but for the time being it stays with turtles/kaas for now until the real ownership is decided.